server/migrations/migration5.js

```diff
@@ -0,0 +1,52 @@
+Migrations.add({
+    version: 5,
+    name: 'mark questions closed/opened',
```

Comment:
- I would rather whether questions should be opened/closed to be moved to collection helpers instead of here .
since it should be more dynamic (it could be computed easily ).. `which also means that you save a migration`
- you don't need a for loop of find command to get objects .. you can use [$in](https://docs.mongodb.com/manual/reference/operator/query/in/) instead.